coala_quickstart/generation/Utilities.py

```diff
@@ -0,0 +1,41 @@
+import os
+from collections import defaultdict
+
+from coala_utils.Extensions import exts
```

Comment:
this is a constant, right? File an issue to name it with more meaning and upper case